src/components/pagination/bl-pagination.test.ts

```diff
@@ -221,7 +221,7 @@ describe('bl-pagination', () => {
         jumper.value = '5';
       }
 
-      setTimeout(() => jumper?.dispatchEvent(new Event('change')));
+      setTimeout(() => jumper?.dispatchEvent(new Event('change', { bubbles: true })));
```

Comment:
Move this line under of `jumper.value`, this will likely to solve the test error.

Reply:
Do you mean like this?

```ts
      if (jumper) {
        jumper.value = '5';

        setTimeout(() => jumper.dispatchEvent(new Event('change', { bubbles: true })));
      }
```

This doesn't change anything.

Reply:
Indeed, the way we fire change event in pagination was a bit problemmatic. Also input and pagination both uses `bl-change` name and that also creates a confusion. I made some refactors in pagination and added `stopPropogation` for `bl-change` event of internal input to not fire 2 different type of `bl-change` event from pagination component. Please check latest version.